packages/sit-onyx/src/components/OnyxTag/types.ts

```diff
@@ -14,4 +15,27 @@ export type OnyxTagProps = DensityProp & {
    * An icon which will be displayed on the left side of the label.
    */
   icon?: string;
+  /**
+   * Deermines whether the Tag is clickalbe or not.
+   * If a string is provided,it represnts a tooltip label
+   * If an object is provided, it contains a tooltip label and ann optional action icon.
+   */
```

Comment:
Typos and added more details regarding the label:
```suggestion
  /**
   * Determines whether the Tag is clickable or not.
   * If a string is provided, it is used as tooltip label.
   * If an object is provided, it contains the tooltip label and an optional action icon.
   *
   * The label must describe the on-click action, e.g. "Click to remove the tag."
   */
```